sdk/core/azure-core/inc/azure/core/http/curl/curl.hpp

```diff
@@ -76,6 +75,7 @@ namespace Azure { namespace Core { namespace Http {
      */
     bool SSLVerifyPeer = true;
 
+#if defined(BUILD_SSL_OPTION_NO_REVOKE)
```

Comment:
Do we really want to hide certain option values behind compiler flags like this? Can we instead leave the option available, and have it no-op OR throw at runtime instead?

How is the caller supposed to write code against this option now?

Reply:
Also, if we really want to keep this ifdef, it should be around `NoRevoke` only, and not the whole `CurlTransportSSLOptions`.

Reply:
that's the only option, I don't like leaving the options struct empty (pointless).